docs/processing_information.md

```diff
@@ -0,0 +1,60 @@
+# Processing information
+
+## Single-cell and single-nuclei RNA-seq
+
+### Alignment and quantification using alevin-fry
+
+We used [`salmon alevin`](https://salmon.readthedocs.io/en/latest/alevin.html) and [`alevin-fry`](https://alevin-fry.readthedocs.io/en/latest/) to generate gene by cell counts matrices for all single-cell and single-nuclei samples.
+In brief, we utilized [selective alignment](#selective-alignment) to the [`splici` index](#the-splici-index) for all single-cell and single-nuclei samples. 
+
+#### Reference transcriptome index
+
+For all samples, we aligned FASTQ files to a reference transcriptome index referred to as the `splici` index.
+The [`splici` index](https://combine-lab.github.io/alevin-fry-tutorials/2021/improving-txome-specificity/) is built using transcripts from both spliced cDNA and intronic regions.
+Inclusion of intronic regions in the index used for alignment allowed us to capture both reads from mature, spliced cDNA and nascent, unspliced cDNA. 
+Alignment of RNA-sequencing data to an index containing intronic regions has been shown to reduce spuriously detected genes ([He _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.06.29.450377v1), [Kaminow _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1.full#sec-5))
```

Comment:
I would probably use the DOI link for the He et al. citation if we don't think we need to tie it to v1. Since you're linking to a particular section for Kaminow et al., I think this is fine.